interestingness/range.py

```diff
@@ -39,7 +39,7 @@ def parseOptions(arguments):
     return options, int(args[0]), int(args[1]), args[2:] # args[0] is minLoopNum, args[1] maxLoopNum
 
 def interesting(cliArgs, tempPrefix):
-    (options, rangeMin, rangeMax, arguments) = parseOptions(cliArgs)
+    (_options, rangeMin, rangeMax, arguments) = parseOptions(cliArgs)
```

Comment:
I'd change `parseOptions` to not return `options` as part of the tuple instead, since the only call site (here) doesn't use it.